12-say-grpc/backend/vendor/github.com/campoy/justforfunc/13-cgo-flite/flite/flite.go

```diff
@@ -0,0 +1,55 @@
+package flite
+
+// #cgo CFLAGS: -I /usr/include/flite
```

Comment:
This is only linux right? You can include a build constraint for linux.

// +build linux